components/org.wso2.carbon.identity.api.server.configs/org.wso2.carbon.identity.api.server.configs.v1/src/gen/java/org/wso2/carbon/identity/api/server/configs/v1/model/InboundAuthSAML2Config.java

```diff
@@ -36,7 +37,7 @@ public class InboundAuthSAML2Config  {
   
     private List<String> destinationURLs = null;
 
-    private Integer metadataValidityPeriod;
+    private BigDecimal metadataValidityPeriod;
```

Comment:
Let remove this file from the commit. Seems like a previous editor of the api definition hasn't updated the gen code.

Reply:
@ThaminduDilshan has added this code. He will test the fix after merging the PR.